uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/viewmodel/RealTimetableViewModel.kt

```diff
@@ -25,10 +24,13 @@ import kotlinx.coroutines.flow.onEach
 import kotlinx.coroutines.flow.receiveAsFlow
 import kotlinx.coroutines.flow.stateIn
 import kotlinx.coroutines.launch
+import javax.annotation.meta.Exhaustive
```

Comment:
Can you apply `./gradlew :spotlessKotlinApply` ? 🙏

Reply:
Formatted 🙇‍♂️
97e3c02